### DISCUSSION TITLE
Minimap instead of full screen one.

### DISCUSSION THREAD

**Discussion by Darmorque** [diasurgical/DevilutionX#3271] [Ideas]
Hi. Not sure whether it's possible in the first place, but what do you think about adding a feature of scaling down the level map overlay and moving it to the side or either left or right corner of the screen? I've seen some concepts in older posts, but I'm unsure whether anything has been done about it since.

**Comment by AJenbo** [comment]
It's possible, it may or may not happen depending on if some one would want to contribute one.

**Comment by licaon-kter** [comment]
@Darmorque Did you see the one in https://github.com/diasurgical/devilutionX/discussions/1656 ?
